tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -277,6 +277,13 @@ impl<C, N, E, W> Subscriber<C, N, E, W> {
     /// Sets whether or not the formatter emits ANSI terminal escape codes
     /// for colors and other text formatting.
     ///
+    /// When the "ansi" crate feature flag is enabled, ANSI colors are enabled
+    /// by default unless the `NO_COLOR` environment variable is set to
+    /// a non-empty value.  If the `NO_COLOR` environment variable is set to
+    /// any non-empty value, then ANSI colors will be suppressed by default.
+    /// The [`with_ansi`] and [`set_ansi`] methods can be used to forcibly
+    /// enable ANSI colors, overriding any `NO_COLOR` environment variable.
```

Comment:
one last tiny nit: what do you think about adding a link to no_color.org here?
```suggestion
    /// by default unless the [`NO_COLOR`] environment variable is set to
    /// a non-empty value.  If the [`NO_COLOR`] environment variable is set to
    /// any non-empty value, then ANSI colors will be suppressed by default.
    /// The [`with_ansi`] and [`set_ansi`] methods can be used to forcibly
    /// enable ANSI colors, overriding any [`NO_COLOR`] environment variable.
    ///
    /// [`NO_COLOR`]: https://no-color.org/
```